config.yaml

```diff
@@ -86,6 +86,17 @@ buffer:
   # - Focused: Only show input when the buffer is focused
   input_visibility: Always
 
+  # Topic banner settings:
+  topic_banner:
+    # Show topic messages in a topic banner at the topic of a pane instead
+    # of server messages.  Topic changes will still show up as a server
+    # message.
+    # - Default is false.
+    enabled: false
+    # Maximum height of the topic banner.
+    # - Default is 42
+    max_height: 42
```

Comment:
I briefly tried to have it set automatically, but I couldn't see how to get iced to give me the line height of the enclosed text (or to let me render offscreen so I can measure the line height).  I found the results where the max_height doesn't reveal an integer multiple of line height (plus padding) to be less than ideal.  I'll dig into it further.

Reply:
Found `double_pass.rs` to use for calculating the rendered dimensions, such that the configuration can be in lines rather than requiring the user to know/determine the line height.  Might be too many passes to first find a window height percentage via `FillPortion` then the nearest number of lines though?